src/JWT/Serializers/SystemTextJsonSerializer.cs

```diff
@@ -0,0 +1,60 @@
+﻿#if SYSTEMTEXTJSON
+namespace JWT.Serializers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json;
+    using JWT;
+
+    public class SystemTextJsonSerializer : IJsonSerializer
```

Comment:
Sealed

Reply:
I think the another, already existing serializer must be included only if the target framework is lower than netstd2